lib/utils/isMap.js

```diff
@@ -0,0 +1,45 @@
+/* @flow */
+'use strict';
+
+/** @typedef {import('postcss-value-parser').Node} ValueNode */
+
+/**
+ * @param {ValueNode | undefined} valueNode
+ * @returns {boolean}
+ */
+module.exports = function(valueNode /*: Object*/) /*: boolean*/ {
```

Comment:
```suggestion
module.exports = function(valueNode) {
```

We removed Flow in master :)

Reply:
> We removed Flow in master :)

And this is awesome :+1: